.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["backpack"]
+  "presets": ["es2015", "backpack"]
```

Comment:
Why is es2015 needed?

Reply:
I seems, that something was wrong without it, but now all work fine...

Reply:
In case you ever need babel transpilation, don't look further than [babel-preset-env](http://babeljs.io/env), you don't need stag-* and es2015 presets anymore

Reply:
Cool, thank you, a already had question about what preset I should use and not. Your answer solve it. :)